custom_components/addhon/command_catalog_store.py

```
@@ -0,0 +1,67 @@
+        except Exception as error:
+            _LOGGER.debug(
+                "Command catalog cache save failed (%s)", type(error).__name__
+            )
+            return
+        self._saved_generation = snapshot.generation
+
+    async def async_remove(self) -> None:
+        """Remove only this config entry's command-catalog document."""
+        try:
+            await self._store.async_remove()
+        except Exception as error:
```

Ruff (0.16.3)

[warning] Do not catch blind exception: `Exception`

(BLE001)